lib/athlete/widgets/build_long_chart.dart

```diff
@@ -38,6 +38,32 @@ SfCartesianChart buildLongChart(
         width: 2,
         opacity: 1,
         dashArray: <double>[5, 5],
+      ),
+      FastLineSeries<GraphData, DateTime>(
+        name: 'Market Value',
+        dataSource: chartStats.toSet().toList(),
```

Comment:
What is going on over there? converting to a set which removes duplicate values and then converting it back to a list. I thought this was handled somewhere else.

Reply:
removed unnecessary converting